BZAccessControl/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "BZAccessControl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx hardhat compile",
+    "test": "npx hardhat test",
+    "deploy": "npx hardhat run scripts/deploy.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Our main license will be MIT

Reply:
Updated license and other things in package.json